hooks/hookexecution/execution_test.go

```diff
@@ -55,6 +55,9 @@ func TestHandleModuleActivitiesBidderRequestPayload(t *testing.T) {
 			activityControl := privacy.NewActivityControl(test.privacyConfig)
 			newPayload := handleModuleActivities(test.hookCode, activityControl, test.inPayloadData)
 			assert.Equal(t, test.expectedPayloadData.Request.BidRequest, newPayload.Request.BidRequest)
+			//check input payload didn't change
+			origInPayloadData := test.inPayloadData
+			assert.Equal(t, origInPayloadData, test.inPayloadData)
```

Comment:
I think you need to move the line `origInPayloadData := test.inPayloadData` above the call to `handleModuleActivities`.